python/DynamicLandCoverMapping/Dynamic land cover mapping.ipynb

```diff
@@ -305,19 +305,30 @@
     "    # properties to filter the data\n",
     "    props = {\"eo:cloud_cover\": lambda v: v <= 80}\n",
     "\n",
-    "    bands = [\"B03\", \"B04\", \"B05\", \"B06\", \"B07\", \"B08\", \"B11\", \"B12\", \"SCL\"]\n",
+    "    bands = [\"B03\", \"B04\", \"B05\", \"B06\", \"B07\", \"B08\", \"B11\", \"B12\"]\n",
     "    idx_list = [\"NDVI\", \"NDMI\", \"NDRE1\", \"NDRE2\", \"NDRE5\"]\n",
     "\n",
     "    # establish a connection to the data source\n",
     "    c = connection_provider()\n",
+    "    scl = c.load_collection(\n",
+    "        \"SENTINEL2_L2A\", temporal_extent=temp_ext_s2, bands=\"SCL\", properties=props\n",
+    "    )\n",
+    "    cloud_mask = scl.process(\n",
+    "        \"to_scl_dilation_mask\",\n",
+    "        data=scl,\n",
+    "        kernel1_size=17, kernel2_size=77,\n",
+    "        mask1_values=[2, 4, 5, 6, 7],\n",
+    "        mask2_values=[3, 8, 9, 10, 11],\n",
+    "        erosion_kernel_size=3\n",
+    "    )\n",
     "    s2 = c.load_collection(\n",
     "        \"SENTINEL2_L2A\", temporal_extent=temp_ext_s2, bands=bands, properties=props\n",
     "    )\n",
+    "    # apply cloud masking\n",
+    "    s2 = s2.mask(cloud_mask)\n",
     "\n",
-    "    # apply cloud masking and filter out the scl band\n",
-    "    s2 = s2.process(\"mask_scl_dilation\", data=s2, scl_band_name=\"SCL\").filter_bands(\n",
-    "        s2.metadata.band_names[:-1]\n",
-    "    )\n",
+    "    #filter out the scl band\n",
+    "    s2 = s2.filter_bands(s2.metadata.band_names[:-1])\n",
```

Comment:
what happened here is that the scl band was removed from the entire cube. Now however the band should not be in the cube to start with